packages/migrate/src/__tests__/DbExecute.test.ts

```diff
@@ -0,0 +1,702 @@
+
+
+            `)
+    })
+  })
+
+  describe('sqlite', () => {
+    const sqlScript = `-- Drop & Create & Drop
+DROP TABLE IF EXISTS 'test-dbexecute';
+CREATE TABLE 'test-dbexecute' ("id" INTEGER PRIMARY KEY);
+DROP TABLE 'test-dbexecute';`
+
+    it('should pass with --file --schema', async () => {
+      ctx.fixture('schema-only-sqlite')
+
+      fs.writeFileSync('script.sql', sqlScript)
+      const result = DbExecute.new().parse([
+        '--preview-feature',
+        '--schema=./prisma/schema.prisma',
+        '--file=./script.sql',
+      ])
+      await expect(result).resolves.toMatchInlineSnapshot(`Script executed successfully.`)
+    })
+
+    it('should pass using a transaction with --file --schema', async () => {
+      ctx.fixture('schema-only-sqlite')
+
+      fs.writeFileSync(
+        'script.sql',
+        `-- start a transaction
+BEGIN;
+
+${sqlScript}
+
+-- commit changes    
+COMMIT;`,
+      )
+      const result = DbExecute.new().parse([
+        '--preview-feature',
+        '--schema=./prisma/schema.prisma',
+        '--file=./script.sql',
+      ])
+      await expect(result).resolves.toMatchInlineSnapshot(`Script executed successfully.`)
+    })
+
+    it('should pass with --file --url=file:dev.db', async () => {
+      ctx.fixture('introspection/sqlite')
+
+      fs.writeFileSync('script.sql', sqlScript)
+      const result = DbExecute.new().parse(['--preview-feature', '--url=file:dev.db', '--file=./script.sql'])
+      await expect(result).resolves.toMatchInlineSnapshot(`Script executed successfully.`)
+    })
+
+    it('should fail with P1013 error with invalid url with --file --url', async () => {
+      ctx.fixture('schema-only-sqlite')
+      expect.assertions(2)
+
+      fs.writeFileSync('script.sql', '-- empty')
+      try {
+        await DbExecute.new().parse(['--preview-feature', '--url=invalidurl', '--file=./script.sql'])
+      } catch (e) {
+        expect(e.code).toEqual('P1013')
+        expect(e.message).toMatchInlineSnapshot(`
+          P1013
+
+          The provided database string is invalid. Error parsing connection string: relative URL without a base in database URL. Please refer to the documentation in https://www.prisma.io/docs/reference/database-reference/connection-urls for constructing a correct connection string. In some cases, certain characters must be escaped. Please check the string for any illegal characters.
+
+        `)
+      }
+    })
+
+    // TODO, it's passing?
+    //  it('should fail with P1001 error with unreachable url with --file --url', async () => {
+    it('should pass with --file --url=file:doesnotexists.db', async () => {
+      ctx.fixture('introspection/sqlite')
+
+      fs.writeFileSync('script.sql', sqlScript)
+      const result = DbExecute.new().parse(['--preview-feature', '--url=file:doesnotexists.db', '--file=./script.sql'])
+      await expect(result).resolves.toMatchInlineSnapshot(`Script executed successfully.`)
+    })
```

Comment:
Apparently a non-existing SQLite database file doesn't error, I guess it's auto-created. @tomhoule do you see a problem with that?
Maybe it's ok, I think it’s our default SQLite behavior.